test/e2e/harness/src/app/app.module.ts

```diff
@@ -99,7 +99,7 @@ const appRoutes: Routes = [
 // On the callback load we detect PKCE by inspecting the pathname
 const url = new URL(window.location.href);
 const pkce = !!url.searchParams.get('pkce') || url.pathname.indexOf('pkce/callback') >= 0;
-const redirectUri = window.location.origin + (pkce ? '/pkce/callback' : '/implicit/callback');
+const redirectUri = window.location.origin + (pkce ? '/pkce/callback' : '/login/callback');
```

Comment:
If it isn't too time consuming, would be great to remove the `/pkce/callback` logic. It is not needed anymore, and this low level details is out of scope for the Angular SDK. Would be simpler with a single callback, we can still see if it is PKCE if we need to.